src/UI/SDK/src/api/BuyerLocations.ts

```diff
@@ -196,6 +197,12 @@ export default class BuyerLocations {
         return await httpClient.get(`/buyerlocations/${userGroupType}/${parentID}/usergroupassignments/${userID}`, { params: {  accessToken, impersonating } } );
     }
 
+    public async SetUserGroups(buyerID: string, newUserID: string, accessToken?: string): Promise<void> {
+        const impersonating = this.impersonating;
+        this.impersonating = false;
+        return await httpClient.put(`/buyerlocations/${buyerID}/transferanonymous/${newUserID}`, undefined, { params: { accessToken, impersonating } } )
+    }
```

Comment:
Can you choose a single name for all of these actions? I think if they're all ultimately pointing to the same piece of code they should be consistently named

The sdk method is called SetUserGroups
The endpoint is called TransferAnonymous
The method in the controller is ApplyAnonUserGroupsToNewUser
The service method in the backend is called AssignNewUserAnonUserGroups

Reply:
Just updated.